Altis_Life.Altis/config/Config_Clothing.hpp

```diff
@@ -148,13 +165,14 @@ class Clothing {
             { "G_Lady_Blue", "", 150, "" }
         };
         vests[] = {
-            { "NONE", "Remove Vest", 0, "" },
+            { "NONE", $STR_C_Remove_vests, 0, "" },
             { "V_Rangemaster_belt", "", 800, "" },
             { "V_TacVest_blk_POLICE", "", 1000, "call life_coplevel >= 1" },
-            { "V_PlateCarrier2_rgr", "", 1500, "call life_coplevel >= 2" }
+            { "V_PlateCarrier2_rgr", "", 1500, "call life_coplevel >= 2" },
+            { "V_TacChestrig_grn_F", "", 1500, "call life_coplevel >= 2" } //Apex DLC
```

Comment:
[`V_TacChestrig_grn_F`](https://community.bistudio.com/wiki/Arma_3_CfgWeapons_Equipment#V_TacChestrig_grn_F) looks more appropriate for the rebels.